data/brands/amenity/bank.json

```diff
@@ -14930,6 +14930,28 @@
         "name:en": "Kagoshima Bank",
         "name:ja": "鹿児島銀行"
       }
+    },
+    {
+      "displayName": "АИК банка",
+      "locationSet": {"include": ["rs"]},
+      "matchNames": [
+        "AIK bank",
+        "AIK banka",
+        "AIK BANK",
+        "AIK BANKA",
+        "АИК банк",
+        "АИК БАНК"
+      ],
+      "tags": {
+        "amenity": "bank",
+        "brand": "АИК банка",
+        "brand:wikidata": "Q293650",
+        "website": "https://www.aikbanka.rs/",
```

Comment:
```suggestion
```

Sorry, NSI don't collect website data. This had documented in https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required
Such detailed data can be sent to Wikidata. So I'll remove this line.